src/app/docs/[docId]/evaluations/EvaluationsClient.tsx

```diff
@@ -40,6 +40,18 @@ export default function EvaluationsClient({
   const [agentsWithEvaluations, setAgentsWithEvaluations] = useState<
     AgentWithEvaluation[]
   >([]);
+
+  // Get current document version from evaluation data
+  const currentDocumentVersion = React.useMemo(() => {
```

Comment:
why are we doing this? At least, can we do this in the server layer? Like, "isCurrent"? 

Do you think this should be another db field?